ggplot/qplot.py

```diff
@@ -7,13 +7,11 @@
 from .geoms import geom_point, geom_bar, geom_histogram, geom_line # , geom_boxplot
 from .scales.scale_log import scale_x_log, scale_y_log
 import pandas as pd
-import numpy as np
 import six
 
-
 def qplot(x, y=None, color=None, size=None, fill=None, data=None,
-        geom="auto", stat=[], position=[], xlim=None, ylim=None, log="",
-        main=None, xlab=None, ylab="", asp=None):
+          geom="auto", stat=[], position=[], xlim=None, ylim=None, log="",
```

Comment:
Could you fix the indentation on these two lines?

Reply:
I think it's fixed (looks fine in my emacs) although if it's still showing up wrong for you I'll fix in the browser

Reply:
Hey! Yeah it still seems to be off :/

Reply:
Okay I see, do you want the new lines consistent with how it was before?